autobahn_autoreconnect/__init__.py

```diff
@@ -92,7 +92,7 @@ class ApplicationRunner(object):
     """
 
     def __init__(self, url, realm, extra=None, serializers=None, debug_app=False,
-                 ssl=None, loop=None, retry_strategy=BackoffStrategy()):
+                 ssl=None, loop=None, retry_strategy=BackoffStrategy(), autoPingInterval=10, autoPingTimeout=27):
```

Comment:
One last nit: Should use snake case (`auto_ping_interval`) for consistency with other parameters. Thanks!